### PR TITLE
Migrate to GitHub CI

### PR DESCRIPTION
**Pull request by kethinov**
Before merging:

Add the `CODECOV_TOKEN` token to Secrets of this repo: Settings -> Secrets -> Add  `CODECOV_TOKEN` (get the token contents from Codecov and paste it)

The build won't start until this is merged.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
   ],
   "author": "JP Richardson <jprichardson@gmail.com>",
   "contributors": [
-    "Alexander J. Lallier <alexanderlallier@aol.com>"
+    "Alexander J. Lallier <alexanderlallier@aol.com>",
+    "Eric Newport <kethinov@gmail.com>"
```
Comment (alallier) [inline package.json]:
```suggestion-1
"Alexander J. Lallier <alexanderlallier@aol.com>"
```

Eric just want to keep contributors list to added collaborators, this is inline with what we researched around npm with when the project was turned over to me and it helps keep down `package.json` sprawl

File: package.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
   ],
   "author": "JP Richardson <jprichardson@gmail.com>",
   "contributors": [
-    "Alexander J. Lallier <alexanderlallier@aol.com>"
+    "Alexander J. Lallier <alexanderlallier@aol.com>",
```
Comment (alallier) [inline package.json]:
```suggestion+1
    "Alexander J. Lallier <alexanderlallier@aol.com>"
```

Comment (alallier) [inline package.json]:
```suggestion
    "Alexander J. Lallier <alexanderlallier@aol.com>"
```

**Comment by alallier** [comment]
This looks great, thanks for the contribution! I have the secret added...time to press the magic button and see what happens

**Comment by kethinov** [comment]
Hopefully nothing breaks...

**Comment by alallier** [comment]
No worries if it does, we can touch it up quick

**Comment by alallier** [comment]
It's dying in Windows

**Comment by kethinov** [comment]
Some kind of path issue.

**Comment by kethinov** [comment]
I wonder if `nyc` needs to be `./node_modules/.bin/nyc`

**Comment by alallier** [comment]
Let me try this in a local powershell

**Pull request merged**